awx/main/tests/functional/api/test_inventory.py

```diff
@@ -412,11 +412,16 @@ def test_delete_inventory_host(delete, host, alice, role_field, expected_status_
 # See companion test in tests/functional/test_rbac_inventory.py::test_inventory_source_update
 @pytest.mark.parametrize("start_access,expected_status_code", [(True, 202), (False, 403)])
 @pytest.mark.django_db
-def test_inventory_update_access_called(post, inventory_source, alice, mock_access, start_access, expected_status_code):
+def test_inventory_update_access_called(post, discover_available_cloud_provider_plugin_names, alice, mock_access, start_access, expected_status_code):
```

Comment:
Pytest arguments are coming from the same-named fixtures. There is no fixture with this name, so it'll just fail attempting to populate this argument.
It looks like there should be a fixture called `inventory_source`, though. But since it's not declared in this module, I'd look it up inside a `conftest.py`.